### PR TITLE
Make Tagit labels clickable links for id.kb.se keywords

### PR DESCRIPTION
**Pull request by lrosenstrom**

### CODE REVIEW

File: IdkbsePlugin.inc.php [hunk 1]
```diff
@@ -92,13 +92,28 @@ function( data ) {
...
 										}));
 							}	
 							
 						});
+				},
+	
+				beforeTagAdded: function(event, ui) {
+				var labelSpan = ui.tag[0].childNodes[0];
+				var id = ui.tagLabel;
+				var prefLabel = decodeURIComponent(ui.tagLabel.split(\"/\").pop());
```
Comment (olovy) [inline IdkbsePlugin.inc.php]: Move inside `if` statement below?

**Review by olovy — CHANGES_REQUESTED** [review]
I think we need to get prefLabel from the data (and not from the URI) since e.g. SSIF terms won't have the prefLabel in the URI.

What happens to `label` and `value` returned from `tagSource`? Aren't they avaliable to use in `beforeTagAdded`?

**Comment by lrosenstrom** [comment]
'value' is available in beforeTagAdded, but the variable name is 'tagLabel', so id = ui.tagLabel gets the value of 'value' in tagSource. 
We can put both preflabel and the uri in 'value' and parse the preflabel. In that way we can be sure the preflabel is always correct. The downside with such an approach is that the keywords will show up as 'preflabel|uri' in other views in OJS, and that the OAI-PMH plugin will need more special handling for keyword forms: instead of just dealing with keywords in forms of different URIs it will also need to handle the preflabel/ special characters / uri combo that is specific to this plugin. But I guess we have to go down that path to get the preflabel part right!

**Comment by lrosenstrom** [comment]
Updated the PR after feedback. Another plus with the new approach is that browsing the autocompleted keywords with the keyboard is clearer now that you see the prefLabel before the URI.

**Review by olovy — APPROVED** [review]
LGTM!

File: IdkbsePlugin.inc.php [hunk 2]
```diff
@@ -92,13 +92,29 @@ function( data ) {
...
+					var id = splitValue.pop();
+					
+					var labelSpan = ui.tag[0].childNodes[0];
+					labelSpan.textContent = '';
+					var link = document.createElement(\"a\");
+					link.setAttribute(\"href\", id);
+					link.setAttribute('target', \"_blank\")
+					link.innerHTML = prefLabel;
+					labelSpan.appendChild(link);
 				}
```
Comment (olovy) [inline IdkbsePlugin.inc.php]: Indent one step?

**Pull request merged**